### ISSUE TITLE
No module named 'StringIO'

### ISSUE THREAD

**Issue by nueverest** [theopolisme/webcam-streamer#10]
I'm using Python 3.6.1 and am getting a ModuleNotFoundError.

```bash
import StringIO
ModuleNotFoundError: No module named 'StringIO'
```

It seems the solution is:

```python
try:
    from StringIO import StringIO
except ImportError:
    from io import StringIO
```

http://stackoverflow.com/a/18284900/1783439

**Comment by slivingston** [comment]
I agree with your solution. The same patch was applied to launcher.py in PR #2, but it still needs to be applied other places in the package.

**Comment by carlthome** [comment]
Status on issue? `pip3 install webcam-streamer; webcam-streamer` doesn't work due to this atm.

**Comment by CHURLZ** [comment]
Make the same changes in streamer/__init__.py and it works, but I'm stuck on the next import atm... I'll PR the change once it's running!
